pandas/tests/tools/test_to_datetime.py

```diff
@@ -1490,23 +1465,26 @@ def test_unit_with_numeric(self, cache, errors, dtype):
         tm.assert_index_equal(result, expected)
 
     @pytest.mark.parametrize(
-        "exp, arr",
+        "exp, arr, warning",
         [
             [
                 ["NaT", "2015-06-19 05:33:20", "2015-05-27 22:33:20"],
```

Comment:
Why does this issue a warning again? This is a consistent format no?

Reply:
Thanks! 

This one's fine,  it's the second one (`arr`) which isn't, because of the `'foo'` string